scripts/training.py

```diff
@@ -200,18 +201,11 @@ def optimize_model():
     transitions = memory.sample(parameters["batch_size"])
     batch = Transition(*zip(*transitions))
     non_final_mask = torch.tensor(
-        tuple(
-            map(
-                lambda s: s is not None,
-                batch.next_state
-            )
-        ),
+        tuple(s is not None for s in batch.next_state),
```

Comment:
I am impressed that it folded the map call into generator expression.

Reply:
Sourcery never fails to amaze :)